source/api/presentation/3.0/change-log.md

```diff
@@ -1,125 +1,263 @@
+
+The `description` property was renamed to `summary` and the semantics changed to be a short descriptive text. The new property is a replacement for the content in `metadata`, for situations when a a table of label/value pairs is too much to render. Descriptions that are not short descriptive text are now recorded in `metadata` to allow for appropriate labeling. See issue [#1242](https://github.com/IIIF/api/issues/1242) and entry [1.3.1][prezi30-changelog-long-texts] below.
+
+#### 1.2.7. Rename `related` to `homepage`
+
+Similar to the above changes where non-actionable links and content are put into `metadata`, the `related` property was renamed to `homepage` with more specific semantics of being the home page of the resource. Links to other resources instead go in to `metadata` as HTML. As the intent is to be able to allow `homepage` specific UI affordances, such as an icon, the cardinality was reduced to zero or one. See issues [#1286](https://github.com/IIIF/api/issues/1286) and [#1484](https://github.com/IIIF/api/issues/1484).
+
+#### 1.2.8. Rename `contentLayer` to `supplementary`
+
+With the removal of the `Layer` class in favor of the standard `AnnotationCollection`, and the introduction of the `motivation` value `supplementing`, the `contentLayer` property was renamed to `supplementary`. This conveys that the Annotations in the Annot ation Collection are those with the `motivation` value `supplementing` while avoiding the use of the defunct class name. See issues [#1480](https://github.com/IIIF/api/issues/1480) and [#1174](https://github.com/IIIF/api/issues/1174).
+
+#### 1.2.9. Rename `startCanvas` to `start`, allow reference to part of a Canvas
+
+Following the pattern of removing class names from property names when unnecessary, combined with the change that the referenced resource might be part of a Canvas rather than the entire Canvas, the property was renamed. The increased scope for parts of Canvases is to allow jumping to a specific time point within a Canvas with a `duration` property. See [#1320](https://github.com/IIIF/api/issues/1320).
+
+#### 1.2.10. Rename `within` to `partOf`
+
+While renaming properties, `within` was renamed to `partOf` to follow the same naming convention in ActivityStreams and the Web Annotation Data Model. See issue [#1482](https://github.com/IIIF/api/issues/1482).
+
+### 1.3. Property Value Changes
+
+#### 1.3.1. Allow long texts in `metadata` values
+{: #long-texts}
+
+Several use cases were raised for long texts that were not descriptions, such as bibliography citation lists or provenance histories for artworks. These texts did not have a home in the previous versions, as `metadata` only allowed short values and `description` was a description of the object not information about it. The adopted solution was to allow long texts within `metadata` values, also allowing `description` to be made more specifically a short textual summary. See issue [#1270](https://github.com/IIIF/api/issues/1270)
```

Comment:
This should remind the reader that `description` has been renamed, how about:

The adopted solution was to allow long texts within `metadata` values, and to replace `description` with `summary` for a more specific textual summary.

I know it repeats the information that summary replaces description but it avoids a minor potential confusion.